.github/workflows/convert.yml

```diff
@@ -144,13 +144,26 @@ jobs:
           token: ${{ secrets.IETF_RIBOSE_CI_PAT }}
           path: svn-git-maps
 
+      - name: Tractive - Install tractive
+        run: |
+          sudo gem install tractive
+
       - name: Copy ${{ matrix.svn_repo }}.fo to svn-git-maps
         working-directory: svn-git-maps
         run: |
           git remote set-url origin \
             git@github.com:ietf-ribose/svn-git-maps.git
           cp ${GITHUB_WORKSPACE}/conversion/${{ matrix.svn_repo }}/${{ matrix.svn_repo }}.fo .
 
+      - name: Convert ${{ matrix.svn_repo }}.fo to ${{ matrix.svn_repo }}-revmap.txt
+        working-directory: svn-git-maps
+        run: |
+          tractive generate-revmap \
+            --svn-url https://svn.ietf.org/svn/tools/${{ matrix.svn_repo }}/ \
```

Comment:
`${GITHUB_WORKSPACE}/conversion/${{ matrix.svn_repo }}/${{ matrix.svn_repo }}-mirror` is the local SVN repo, so we don't need to use `--svn-url`.

From `svn-git-maps`, the line would be changed to:
```
--svn-local-path ../conversion/${{ matrix.svn_repo }}/${{ matrix.svn_repo }}-mirror
```